src/pages/builders/selector.ts

```diff
@@ -80,13 +80,18 @@ export default function genSelectors(element: HTMLElement | null) {
   ]);
 
   // We won't use an id selector if the id is invalid (starts with a number)
-  const idSelector =
-    isAttributesDefined(element, ['id']) && !isCharacterNumber(element.id?.[0])
-      ? // Certain apps don't have unique ids (ex. youtube)
-        finder(element, {
-          attr: (name) => name === 'id',
-        })
-      : null;
+  let idSelector = null;
+  try {
+    idSelector =
+      isAttributesDefined(element, ['id']) &&
+      !isCharacterNumber(element.id?.[0])
+        ? // Certain apps don't have unique ids (ex. youtube)
+          finder(element, {
+            attr: (name) => name === 'id',
+          })
+        : null;
+  } catch (e) {}
```

Comment:
we should probably figure why the exception was thrown and prevent that from happening

Reply:
yeah I assume it's the finder package preferring id selectors above anything else, but when it does a uniqueness check it fails that and there's no further logic to fall back to other selectors. I didn't really want to dive into the heuristics code they have built there though 😅